atlp-rwanda/atlp-devpulse-fn
# Login with Google FN

`What does this PR do?`
Login with Google FrontEnd

`Description of Task to be completed?`
Login with google email to access the platform. If the user is not registered in database, he/she will be registered.

`How should this be manually tested?`

- Clone the repo or Pull the changes by `git pull` if you have cloned the repo already.
- Checkout to branch `ft-loginWithGoogle-fn`
- Install all packages by running `npm install`
- Add `CLIENT_ID` in dotenv file obtained from the google client.
- Start app by running npm run dev

`Note`

- Before starting the app make sure the backend is merged.
- The access_token is saved on the local storage.

![Log1](https://user-images.githubusercontent.com/86821764/210132471-59de6a3e-37aa-4250-8c6d-a1438afc3634.PNG)
![Log2](https://user-images.githubusercontent.com/86821764/210132475-23611fd9-24bd-4795-9b9d-2f631eb4a356.PNG)
![log3](https://user-images.githubusercontent.com/86821764/210132479-fbef91a8-c8fc-4b27-a81e-fd219f6e9c38.PNG)
